src/components/Section/History/Employment/EmploymentSummaryProgress.jsx

```diff
@@ -4,43 +4,18 @@ import PropTypes from 'prop-types'
 import { i18n } from 'config'
 
 import SummaryProgress from 'components/Section/History/SummaryProgress'
-import { totalYears } from 'components/Section/History/helpers'
 import { Svg } from 'components/Form'
 
-const dateRangeList = (items, errors) => {
-  const dates = []
-
-  items.forEach((i) => {
-    if (!i.Item) return
-
-    const itemErrors = errors && errors.filter(e => e.indexOf(i.uuid) > -1)
-    if (itemErrors && itemErrors.length > 0) return
-
-    dates.push(i.Item.Dates)
-  })
-
-  return dates
-}
-
 const EmploymentSummaryProgress = (props) => {
-  const {
-    Employment, Birthdate, years, errors,
-  } = props
-
-  let employmentDates = []
-  if (Employment && Employment.List && Employment.List.items) {
-    employmentDates = Employment.List.items
-  }
-
-  const getEmploymentDates = () => dateRangeList(employmentDates, errors)
+  const { dates, years } = props
 
   return (
     <SummaryProgress
       className="residence"
-      List={getEmploymentDates}
+      List={() => dates}
```

Comment:
This might be a little nit since the code was inherited this way. Is there any reason why this `prop` is a function? Can we just invoke the function and pass the array into the component?